AutomatedTesting/Gem/PythonTests/AWS/common/constants.py

```diff
@@ -0,0 +1,19 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+"""
+
+# ARN of the IAM role to assume for retrieving temporary AWS credentials
+ASSUME_ROLE_ARN = 'arn:aws:iam::645075835648:role/o3de-automation-tests'
+# Name of the AWS project deployed by the CDK applications
+AWS_PROJECT_NAME = 'AWSAUTO'
+# Region for the existing CloudFormation stacks used by the automation tests
+AWS_REGION = 'us-west-2'

```

Comment:
nit: Are we fixed to us-west-2? If you changed this what else needs to change as it seems like we are assuming us-west-2 in a lot of places. Reason we don't pick this up from the environment/AWS cli settings/ param passed in?

Reply:
Our automation tests will read the region from this constant. Users can run tests in a different region by updating this constant. I'm checking with infra whether we can read this from environment variable on Jenkins.

Reply:
Created a follow up task to read AWS project name, region and assume role ARN from environment variables